### PR TITLE
docs: translate `checking cluster registration` to chinese

### LINKED ISSUE #8: Chinese version of checking cluster registration
翻译源: https://github.com/clusternet/website/blob/main/content/en/docs/tutorials/cluster-management/checking-cluster-registration.md
翻译后: `content/zh-cn/docs/tutorials/cluster-management/checking-cluster-registration.md`

### PR DESCRIPTION
**Pull request by mangoGoForward**
Signed-off-by: mango <xu.weiKyrie@foxmail.com>

Fixes #8

### CODE REVIEW

**Comment by mangoGoForward** [comment]
@dixudx please have a review when you have free time

**Review by dixudx — COMMENTED** [review]
We may need to rephrase the translations. Thanks for your contribution.

File: content/en/docs/tutorials/cluster-management/checking-cluster-registration.md [hunk 1]
```diff
@@ -89,16 +87,14 @@ rules:
       - '*'
 ```
 
-## Check ManagedCluster Status
+## 检查 ManagedCluster 状态
 
-For every registered cluster that gets approved, there will be a dedicated namespace created for this
-cluster. `ManagedCluster` will be created as well to represent the cluster, including cluster metadata, cluster healthy
-heartbeat, etc.
+每个获得批准的注册集群，都会在此集群创建一个专用的命名空间。创建 `ManagedCluster` 来表示集群，其中包括集群元数据、集群健康心跳等。
```
[the remark this answers is not in the document]
Comment (mangoGoForward) [reply content/en/docs/tutorials/cluster-management/checking-cluster-registration.md]: Done.

File: content/en/docs/tutorials/cluster-management/checking-cluster-registration.md [hunk 2]
```diff
@@ -34,9 +34,7 @@ status:
   token: REDACTED
 ```
 
-After `ClusterRegistrationRequest` is approved, the status will be updated with corresponding credentials that can be
-used to access parent cluster if needed. Those credentials have been set with scoped RBAC rules, see blow two rules for
-details.
+`ClusterRegistrationRequest` 获准后，用于访问父集群的相应凭据的状态将被更新。这些凭据使用集群或项目空间 RBAC 规则进行设置，有关详细信息，请参阅以下两条规则。
```
Comment (dixudx) [inline content/en/docs/tutorials/cluster-management/checking-cluster-registration.md:37]: Still obscure.

**Review by dixudx — COMMENTED** [review]
We're getting close. But there are still some sentences we need to improve.

File: content/en/docs/tutorials/cluster-management/checking-cluster-registration.md (hunk 2 above)
Comment (mangoGoForward) [reply content/en/docs/tutorials/cluster-management/checking-cluster-registration.md]: Thanks, already update, can you have a review?

**Review by dixudx — APPROVED** [review]
Nice work!

**Pull request merged**